.github/workflows/test.yml

```diff
@@ -127,6 +127,7 @@ jobs:
         github-token: ${{ secrets.WEAVE_GITOPS_TEST_WEAVEWORKS_WEAVE_GITOPS_BOT_TOKEN }}
         gitlab-key: ${{ secrets.GITLAB_KEY }}
         gitlab-token: ${{ secrets.GITLAB_TOKEN }}
+        artifacts-base-dir: "/tmp/gitops-test"
```

Comment:
This value is used in a whole bunch of jobs. I think it should be moved to the environment for the workflow:
```
name: run tests
env:
  ARTIFACTS_BASE_DIR: "/tmp/gitops-test"
jobs:
...
       artifacts-base-dir: ${{ env.ARTIFACTS_BASE_DIR }}
...
```